### PR TITLE
add possibility to view avatars fullscreen

### PR DESCRIPTION
**Pull request by connyduck**
Closes https://github.com/tuskyapp/Tusky/issues/669

Not my prettiest code, but does the job.
I would love to have a ripple effect on the avatar, but no idea how to achieve that 🤔

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/AccountActivity.kt [hunk 1]
```diff
@@ -321,6 +322,15 @@ class AccountActivity : BottomSheetActivity(), ActionButtonActivity, HasSupportF
                     .load(account.header)
                     .into(accountHeaderImageView)
 
+            accountAvatarImageView.setOnClickListener {
+                val intent = ViewMediaActivity.newAvatarIntent(it.context, account.avatar)
+
+                ViewCompat.setTransitionName(it, account.avatar)
```
Comment (charlag) [inline app/src/main/java/com/keylesspalace/tusky/AccountActivity.kt]: IMHO this lambda is too big to use `it`, it takes some effort to understand what it means

File: app/src/main/java/com/keylesspalace/tusky/AccountActivity.kt [hunk 2]
```diff
@@ -321,6 +322,15 @@ class AccountActivity : BottomSheetActivity(), ActionButtonActivity, HasSupportF
...
+                val options = ActivityOptionsCompat.makeSceneTransitionAnimation(this@AccountActivity, it, account.avatar)
```
Comment (charlag) [inline app/src/main/java/com/keylesspalace/tusky/AccountActivity.kt]: You don't need a `this` qualifier here, if I understand correctly

**Comment by connyduck** [comment]
thx @charlag

**Comment by moshpirit** [comment]
In what release will we see this feature? it's not in F-droid v. 2.3

**Comment by connyduck** [comment]
It will be in Tusky 3. You can try it in the nightly right now: https://play.google.com/store/apps/details?id=com.keylesspalace.tusky.test

**Comment by moshpirit** [comment]
Great! Thanks a lot!

**Pull request merged**